Frameworks/CoreGraphics/CGGraphicBufferImage.mm

```diff
@@ -155,8 +157,7 @@
     if (_renderTarget == nullptr) {
         BYTE* imageData = static_cast<BYTE*>(LockImageData());
         ComPtr<IWICBitmap> wicBitmap = Make<CGIWICBitmap>(this, SurfaceFormat());
-        ComPtr<ID2D1Factory> d2dFactory;
-        THROW_IF_FAILED(D2D1CreateFactory(D2D1_FACTORY_TYPE_SINGLE_THREADED, __uuidof(ID2D1Factory), &d2dFactory));
+        ComPtr<ID2D1Factory> d2dFactory = _GetD2DFactoryInstance();
```

Comment:
>_GetD2DFactoryInstance [](start = 42, length = 22)

This might be going cross-module soon; perhaps it should be named _CGGetD2DFactory(), and have the signature

```
HRESULT _CGGetD2DFactory(ID2D1Factory** pFactory);
```